src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -32,12 +32,14 @@
 import javax.swing.*;
 import javax.swing.tree.DefaultMutableTreeNode;
 import javax.swing.tree.DefaultTreeCellRenderer;
+import javax.swing.tree.TreeCellRenderer;
 import javax.swing.tree.TreePath;
 import javax.xml.parsers.ParserConfigurationException;
 import java.awt.*;
 import java.awt.event.KeyAdapter;
 import java.awt.event.KeyEvent;
 import java.awt.event.MouseEvent;
+import java.awt.event.*;
```

Comment:
Instead of importing all `java.awt.event` packages, it would be better to just import the one that is needed here, which is `java.awt.event.MouseMotionAdapter`

Reply:
done.